bin/sozo/src/commands/migrate.rs

```diff
@@ -43,6 +43,9 @@ pub enum MigrateCommand {
     Apply {
         #[command(flatten)]
         transaction: TransactionOptions,
+        #[arg(long)]
+        #[arg(help = "Verify classes declared during migration.")]
+        walnut: bool,
```

Comment:
this will cause conflicts with the same arg in `TransactionOptions`. im assuming both flags are meant to do the same thing and considering the latter is already set as a global arg, then maybe we don't need gto specify it here?